### ISSUE TITLE
ability to see your workspace before talking to the LLM

### ISSUE THREAD

**Issue by CitroenGames** [All-Hands-AI/OpenHands#5859]
**What problem or use case are you trying to solve?**

lets say you want to clone a project from gitbucket because openhands doesnt support other git platforms i have to start a conversation and then stop the conversation to get access to the workspace.

**Describe the UX of the solution you'd like**

a button to create a new empty workspace.

**Do you have thoughts on the technical implementation?**

no :D

**Describe alternatives you've considered**

there is none because i dont want other git platforms implented i just want access to the workspace before i can talk to the LLM

**Comment by mamoodi** [comment]
In your scenario are you using your local filesystem where you have the project cloned? And you have to first give a prompt just to go to the next page?

**Comment by CitroenGames** [comment]
> In your scenario are you using your local filesystem where you have the project cloned? And you have to first give a prompt just to go to the next page?

no i dont use local filesystem. yes you do have to interact with the UI to tell the LLM what you want before seeing the workspace UI

**Comment by CitroenGames** [comment]
in devin you could set the whole workspace up and also it would save the workspace for all future prompts. and you can select what container you want to use for that prompt. making it more like deving would be a good idea in my opinion.

**Comment by mamoodi** [comment]
Yeah makes sense. I guess I'm trying to understand at what stage does OpenHands get access to the project. If it's not a github project, you give it a url to clone the project, right?

**Comment by CitroenGames** [comment]
yeah you could but it would be easier to just have a button to open a workspace and let me do all of the setup work.

**Comment by BradKML** [comment]
Seconding this issue, but for seeing if the local folder is loaded properly in the UX before picking a task prompt.
